yamlenv/env.py

```diff
@@ -79,5 +80,5 @@ def interpolate(data):
             x = data
             for k in path[:-1]:
                 x = x[k]
-            x[path[-1]] = e.value
+            x[path[-1]] = yaml.load(e.value)
```

Comment:
Should this be `yaml.safe_load`, for added security?

Reply:
I wonder if it's cleaner to have `EnvVar` expose an attributed like `yaml_value` that just calls `yaml.safe_load`: `interpolate()` at the moment seems to be doing too many things.

Reply:
> Should this be `yaml.safe_load`, for added security?
I don't personally mind, although you might then want to consider the use of `safe_load` in `yamlenv.Loader.construct_include` for consistency ske.